content/uk/docs/reference/glossary/event.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Подія
```

Comment:
Event is an API kind; I'd leave it untranslated.

Reply:
👍 

May I ask you to make a suggestion here for an easier text change?